pkgs/development/libraries/cxxtest/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, python, pythonPackages}:
+
+stdenv.mkDerivation rec {
+  version = "4.4";
+  name = "cxxtest";
+
+  src = fetchFromGitHub {
+    owner = "CxxTest";
+    repo = name;
+    rev = version;
+    sha256 = "19w92kipfhp5wvs47l0qpibn3x49sbmvkk91yxw6nwk6fafcdl17";
+  };
+
+  buildInputs = with pythonPackages; [ python wrapPython ];
+
+  installPhase = ''
+    cd python
+    python setup.py install --prefix=$out
```

Comment:
The python package is used in the main script "cxxtestgen". This package is very specific so I don't think it can be useful in another application.